src/app/components/source/Source.js

```diff
@@ -173,6 +173,105 @@ const SourceContainer = Relay.createContainer(SourceComponent, {
               }
             }
           },
+          log(first: 10000) {
```

Comment:
no it should be under `source` to list all activities across all `project_source`s https://mantis.meedan.com/view.php?id=6174#c22367